template.yaml

```diff
@@ -548,8 +728,53 @@ Resources:
         !Sub
         - |-
           {
-            "StartAt": "ShouldRestoreClusterOrInstance",
+            "StartAt": "IsTaskRestoreInTargetAccountsAndRegions",
```

Comment:
Will this change the json input as well ?

Reply:
json input will have additions if this task type is chosen. otherwise it will remain the same

Reply:
ok lets include in readme and drop a note to tom to blog this capability out

Reply:
this is an incremental PR - i will add some more changes before this is complete. this currently works only for db instance and not cluster